activelearning/dataset/grid.py

```diff
@@ -13,6 +15,8 @@ class GridData(Data):
         y_data: array(N, 1); scores at each position
         normalize_scores: bool; maps the scores to [0; 1]
         grid_size: int; specifies the width and height of the grid (grid_size x grid_size); used to normalize the state positions
+        state2result: function that takes raw states (X_data) and transforms them into the desired format;
```

Comment:
What format is "raw states"? (sorry I'm being picky here but it's for a good reason). Confusion with data representations has been the source of a lot of pain and wasted time and I would like to avoid this in the future.